### DISCUSSION TITLE
[v4] Do you still accept bug reports and pull requests for adonisjs 4.1

### DISCUSSION THREAD

**Discussion by maxgalbu** [adonisjs/core#2327] [General]
Seeing that many of us are still using adonisjs 4.1, do you have a plan to maintain and accept pull requests for adonis 4.1 packages? Eg I think I found an issue in https://github.com/adonisjs/auth, but there's no branch with 4.1 code, only tags. adonisjs/ally was easier since 4.1 code was on master, but other packages?

**Comment by thetutlage** [comment]
@maxgalbu Yes. During the v5 main release, the AdonisJS 4.1 will also receive an overhaul update for all the bug fixes and the docs will also be available. So feel free to create PRs for the bugs you can fix :)

**Reply by koficypher** [reply]
Just a reminder that v4.1 docs cant be found currently. Any help in locating them or they'll be released at a later time ?

**Reply by thetutlage** [reply]
They are available here https://legacy.adonisjs.com/. Mentioned in the release notes as well

**Reply by koficypher** [reply]
Thanks and i have taken note as well.
